test/version-check.js

```diff
@@ -0,0 +1,27 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+//
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2024 Datadog, Inc.
+
+const wt = require('worker_threads')
+
+// Do not want this running on loader thread
+if (wt.isMainThread) {
```

Comment:
Ah, I have muscle memory around linters that don't understand that CJS files are actually function blocks, so generally avoid top-level returns, even if they're warranted. Since the linter here allows it, I'll use it. Update incoming.

Reply:
I guess I've been ignoring such lint errors or just haven't seen them 😅